code/modules/mob/living/basic/guardian/guardian_types/lightning.dm

```diff
@@ -0,0 +1,95 @@
+/// A reasonably durable guardian linked to you by a chain of lightning, zapping people who get between you
+/mob/living/basic/guardian/lightning
+	guardian_type = GUARDIAN_LIGHTNING
+	melee_damage_lower = 7
+	melee_damage_upper = 7
+	attack_verb_continuous = "shocks"
+	attack_verb_simple = "shock"
+	melee_damage_type = BURN
+	attack_sound = 'sound/machines/defib_zap.ogg'
+	damage_coeff = list(BRUTE = 0.7, BURN = 0.7, TOX = 0.7, CLONE = 0.7, STAMINA = 0, OXY = 0.7)
+	range = 7
+	playstyle_string = span_holoparasite("As a <b>lightning</b> type, you will apply lightning chains to targets on attack and have a lightning chain to your summoner. Lightning chains will shock anyone near them.")
+	creator_name = "Lightning"
+	creator_desc = "Attacks apply lightning chains to targets. Has a lightning chain to the user. Lightning chains shock everything near them, doing constant damage."
+	creator_icon = "lightning"
+	/// Link between us and our summoner
+	var/datum/component/summoner_chain
+	/// Associative list of chained enemies to their chains
+	var/list/enemy_chains
+
+/mob/living/basic/guardian/lightning/death(gibbed)
+	. = ..()
+	clear_chains()
+
+/mob/living/basic/guardian/lightning/Destroy()
+	clear_chains()
+	return ..()
```

Comment:
don't we always call `Destroy()` after `death()` on holoparasites since we dust on death (AKA delete ourselves)`. i think this is needless redundancy

Reply:
Better to have and not need than need and not have imo. I leave this kind of redundancy in action code often